packages/picasso/src/Select/types.ts

```diff
@@ -30,6 +30,8 @@ export interface SelectProps<
     > {
   /** If true, the 'Select' will be disabled */
   disabled?: boolean
+  /** To render select options in portal. Should be disabled in Modals */
```

Comment:
Otherwise looks like description for `enablePortal`
```suggestion
  /** Whether to render select options in portal. Should be disabled in Modals */
```